src/log.js

```diff
@@ -198,20 +215,13 @@ class Log extends GSet {
    * @return {Log}   New Log containing the appended value
    */
   async append (data, pointerCount = 1) {
-    // Verify that we're allowed to append
-    if ((this._key && this._key.getPublic)
-        && !this._keys.includes(this._key.getPublic('hex')) 
-        && !this._keys.includes('*')) {
-      throw new Error("Not allowed to write")
-    }
-
```

Comment:
Removing this, and just leaving the validator in, what we'll have is no access control, just a verification that the signature is solid. Any thoughts on how we can check if the entry is actually allowed to be appended?

Reply:
I assumed it'd happen only at the orbit access controller. `ipfs-log` would handle validation (check if the signature is valid) and call the signing function but not if it's allowed to be appended. I thought it'd make sense since permissions are defined at db level. (As a side-note, I think we could change entry creation so the access controller could sign it and `ipfs-log` would only ensure the signature is valid but I don't know if that's ok, and if it is, that's a topic for a future discussion :) )

Reply:
Sorry to keep harping about that if we split up entry creation, could we make the access controller sign entries and then ipfs-log would only be in charge of verifying signatures? (I'm assuming the only problem with that in our previous discussion was that we needed to sign the whole entry object)

Reply:
My understanding was that we would pass the access controller to the log...we can check if appends are permitted by adding 'acl.canAppend(id)' here. The integrity check should be at the log level... I think this is what you're saying above if I understand?

Reply:
Yes, `canAppend` implementation would lie at the `orbit-db` access controller level though and ipfs-log would only have to call the functions it takes from the "validator" (the access controller)

Reply:
Just as an example, this is more or less what I wanna achieve in the end: https://gist.github.com/thiagodelgado111/706529326d488c558442f9055bd67d31